action.yml

```diff
@@ -23,26 +23,11 @@ inputs:
 runs:
   using: composite
   steps:
-    - name: Set up wiki repository
+    - id: index
+      run: '"${GITHUB_ACTION_PATH%/}/src/index.sh"'
       shell: bash
-      working-directory: ${{ inputs.wiki-directory }}
-      run: |
-        git init
-        git config --local user.name github-actions
-        git config --local user.email github-actions@github.com
-        git remote add origin https://${{ inputs.token }}@${{ inputs.repository }}.wiki.git
-        git fetch origin
-        git reset origin/master
-    - name: Commit and push wiki changes
-      shell: bash
-      working-directory: ${{ inputs.wiki-directory }}
-
-      run: |
-        git add --all
-        git commit -m "${{ inputs.commit-message }}"
-        git push -u origin master
-    - name: Cleanup
-      shell: bash
-      working-directory: ${{ inputs.wiki-directory }}
-      run: rm -rf ./.git
-
+      env:
+        INPUTS_REPOSITORY: ${{ inputs.repository }}
+        INPUTS_TOKEN: ${{ inputs.token }}
+        INPUTS_WIKI_DIRECTORY: ${{ inputs.wiki-directory }}
+        INPUTS_COMMIT_MESSAGE: ${{ inputs.commit-message }}
```

Comment:
Nitpick: maybe prefix with something like `AW_` (Actions Wiki) to reduce the (already low) chance of a conflict.

Reply:
it does seem like there is a kind of precedant convention to use `INPUTS_` as a prefix: https://github.com/search?q=path%3A%2Faction.yml+INPUTS_&type=code

but the more popular version seems to be `INPUT_` (no "S") https://github.com/search?q=path%3A%2Faction.yml+INPUT_&type=code

so that might be worth conforming to to fit the existing convention

Reply:
used `INPUT_*` in https://github.com/spenserblack/actions-wiki/pull/35/commits/2b253706be814b61efa2eee780f13e1954ecd1dd

Reply:
Yeah, [`INPUT_`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-of-jobsjob_idstepswith) is the action environment variable name of inputs, so I'll accept that :+1: